new/package.json

```diff
@@ -4,9 +4,13 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "antd": "^3.26.3",
```

Comment:
This is already there in the package.json of client, we do not need it here anymore.

Bundle size would be large otherwise. Remove it from here.